lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -110,6 +110,29 @@ def self.run(params)
 
           Helper::RevenuecatInternalHelper.create_pr(pr_title, body, repo_name, current_branch, new_branch_name, github_pr_token, [label])
         end
+
+        # For older releases than the latest published version, we need to create a new PR against main to update the changelog with the changes from this release
+        is_older_than_latest_published_version = !Helper::RevenuecatInternalHelper.newer_than_latest_published_version?(new_version_number)
+        if is_older_than_latest_published_version
+          
+          if dry_run
```

Comment:
should this be if `!dry_run`?

Reply:
I _think_ `if dry_run` is correct here:

When `dry_run`, the lines above will generate changes that won't be committed. Therefore, the checkout of `main` will fail unless we discard the previous changes.

When `!dry_run`, all the changes above will have been committed and pushed, so there shouldn't be any uncommited changes at this point.

Does it make sense?

In any case, I will add a comment clarifying this, as it can be confusing 👍

Reply:
Hmm I wonder if we should clear these uncommited changes always on dry-run before this... Since it's not directly related to this process... but yeah, a comment works for now.